src/components/common/FormButton/FormButton.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Button } from 'antd';
+
+import './FormButton.styles.css';
+
+const FormButton = props => {
```

Comment:
Just to keep consistency , next time we can destructuring the props. But we will leave this for now